ur'strings' are a Syntax Error in Python 3

You can have __u__'strings' or you can have __r__'strings' but you can not have __ur__'strings' in Python 3.

flake8 testing of https://github.com/scrapy/scrapy on Python 3.6.2

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./docs/conf.py:194:51: E901 SyntaxError: invalid syntax
  ('index', 'Scrapy.tex', ur'Scrapy Documentation',
                                                  ^

./docs/utils/linkfix.py:23:72: E901 SyntaxError: invalid syntax
line_re = re.compile(ur'(.*)\:\d+\:\s\[(.*)\]\s(?:(.*)\sto\s(.*)|(.*))')
                                                                       ^
```

--- a/docs/conf.py
+++ b/docs/conf.py
@@ -191,7 +191,7 @@ htmlhelp_basename = 'Scrapydoc'
 # Grouping the document tree into LaTeX files. List of tuples
 # (source start file, target name, title, author, document class [howto/manual]).
 latex_documents = [
-  ('index', 'Scrapy.tex', ur'Scrapy Documentation',
+  ('index', 'Scrapy.tex', u'Scrapy Documentation',
    ur'Scrapy developers', 'manual'),
 ]
 

--- a/docs/utils/linkfix.py
+++ b/docs/utils/linkfix.py
@@ -20,7 +20,7 @@ _filename = None
 _contents = None
 
 # A regex that matches standard linkcheck output lines
-line_re = re.compile(ur'(.*)\:\d+\:\s\[(.*)\]\s(?:(.*)\sto\s(.*)|(.*))')
+line_re = re.compile(u'(.*)\:\d+\:\s\[(.*)\]\s(?:(.*)\sto\s(.*)|(.*))')
 
 # Read lines from the linkcheck output file
 try:

